### PR TITLE
[css-flex] More changes to test suite after review of files

### PR DESCRIPTION
**Pull request by gsnedders**
This is the changes from #6351 split out into a new PR.

From #6351:

 > negative-margins-001.html: Update this & ref to reflect reality of implementations.

See also the discussion at https://github.com/w3c/web-platform-tests/pull/6351#discussion_r124666811

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19818)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/291571653)

File: css/css-flexbox-1/negative-margins-001.html [hunk 1]
```diff
@@ -4,11 +4,9 @@
 <link rel="author" title="Google Inc." href="https://www.google.com/">
 <link rel="help" href="http://www.w3.org/TR/css-flexbox-1/#item-margins">
 <link rel="match" href="reference/negative-margins-001-ref.html">
-<meta name="assert" content="Tests that for intrinsic size computations,
-an item will not take up less than zero space even with negative margins.">
 <style>
     .container {
-        width: 60px;
+        width: 40px;
```
Comment (gregwhitworth) [inline css/css-flexbox-1/negative-margins-001.html]: Why are you changing this to 40px?

File: css/css-flexbox-1/negative-margins-001.html [hunk 2]
```diff
@@ -4,11 +4,9 @@
 <link rel="author" title="Google Inc." href="https://www.google.com/">
 <link rel="help" href="http://www.w3.org/TR/css-flexbox-1/#item-margins">
 <link rel="match" href="reference/negative-margins-001-ref.html">
-<meta name="assert" content="Tests that for intrinsic size computations,
```
Comment (gregwhitworth) [inline css/css-flexbox-1/negative-margins-001.html]: Why are you deleting this?

**Review by gregwhitworth — DISMISSED** [review]

File: css/css-flexbox-1/negative-margins-001.html (hunk 1 above)
Comment (gsnedders) [reply css/css-flexbox-1/negative-margins-001.html]: That's your change from the PR this is split out of. :)

File: css/css-flexbox-1/negative-margins-001.html (hunk 2 above)
Comment (gsnedders) [reply css/css-flexbox-1/negative-margins-001.html]: Not sure what happened here? Wasn't in the PR this was split out from...

**Comment by gregwhitworth** [comment]
LOL, oh ok - I didn't read the first description (sorry I'm rushing ATM trying to catch up after the holidays). SO really - we need @cbiesinger to take a gander since it was on the Macs that Chrome was different if I recall correctly.

**Comment by foolip** [comment]
Rebased past the directory renaming to get rid of the css-flexbox-1  and status:needs-owners labels.

**Comment by gsnedders** [comment]
does anyone want to review this? this makes the test pass in Chrome, Firefox, and Safari?

**Review by tabatkins — CHANGES_REQUESTED** [review]
Needs the assert fixed (to say the opposite), not just deleted.

After that, LGTM.

**Comment by gsnedders** [comment]
(Someone not me should make that change)

**Review by tabatkins — APPROVED** [review]

**Comment by tabatkins** [comment]
(Thanks, @stephenmcgruer, I was about to make the change but the moment I sent my comment the lunch train rolled thru.)

**Comment by stephenmcgruer** [comment]
@tabatkins I attempted that change, PTAL.

Looking at Chromium source, this test (before this PR) is currently marked as an ignored failure  (https://cs.chromium.org/chromium/src/third_party/blink/web_tests/TestExpectations?l=2346&rcl=46b1cc8c5bf65372f57d12b78233da2292d3c2b7). So I think the change is likely good, and if there's a Chrome Mac bug, then there's a Chrome Mac bug to be filed :)

**Pull request merged**